systemtest/src/test/java/io/strimzi/systemtest/kafka/dynamicconfiguration/DynamicConfSharedST.java

```diff
@@ -40,13 +45,20 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
-/**
- * DynamicConfigurationSharedST is responsible for verify that if we change dynamic Kafka configuration it will not
- * trigger rolling update
- * Shared -> for each test case we same configuration of Kafka resource
- */
 @Tag(REGRESSION)
 @Tag(DYNAMIC_CONFIGURATION)
+@SuiteDoc(
+    description = @Desc("DynamicConfigurationSharedST is responsible for verifying that changing dynamic Kafka configuration will not trigger a rolling update. Shared -> for each test case we use the same Kafka resource configuration."),
```

Comment:
I wonder if the shouldn't add more info to this part as there is no test case doc?

Reply:
I have added test doc to that one test case.